client/oneTimePayment/react/src/context/sdkContext.tsx

```diff
@@ -77,7 +78,7 @@ export const PayPalSDKProvider: React.FC<PayPalSDKProviderProps> = ({
     };
 
     loadPayPalSDK();
-  }, [clientToken, pageType, components]);
```

Comment:
Best I can tell, this "if" statement `if (!sdkInstance && clientToken) {` prevents the existing values in the dependency array from ever causing a new instance to be created. So I figured it was best to just remove them all together and have this useEffect run on every re-render and we can solve for "creating a new sdkInstance when x values change in a future PR". It's something we would want to add a demo for to prove it works as expected. For example, have a select dropdown for changing the pageType.